diff -urN dotproject.orig/classes/date.class.php dotproject/classes/date.class.php
--- dotproject.orig/classes/date.class.php	2003-10-20 08:20:52.000000000 +0000
+++ dotproject/classes/date.class.php	2003-10-20 08:21:14.000000000 +0000
@@ -4,7 +4,7 @@
 * @subpackage utilites
 */
 
-require_once( $AppUI->getLibraryClass( 'PEAR/Date' ) );
+require_once( 'Date.php' );
 
 define( 'FMT_DATEISO', '%Y%m%dT%H%M%S' );
 define( 'FMT_DATELDAP', '%Y%m%d%H%M%SZ' );
diff -urN dotproject.orig/modules/calendar/calendar.class.php dotproject/modules/calendar/calendar.class.php
--- dotproject.orig/modules/calendar/calendar.class.php	2003-10-20 08:20:52.000000000 +0000
+++ dotproject/modules/calendar/calendar.class.php	2003-10-20 08:21:40.000000000 +0000
@@ -3,7 +3,7 @@
 ## Calendar classes
 ##
 
-require_once( $AppUI->getLibraryClass( 'PEAR/Date' ) );
+require_once( 'Date.php' );
 require_once( $AppUI->getSystemClass ('dp' ) );
 
 /**
diff -urN dotproject.orig/modules/projects/projects.class.php dotproject/modules/projects/projects.class.php
--- dotproject.orig/modules/projects/projects.class.php	2003-10-20 08:20:52.000000000 +0000
+++ dotproject/modules/projects/projects.class.php	2003-10-20 08:21:57.000000000 +0000
@@ -6,7 +6,7 @@
 */
 
 require_once( $AppUI->getSystemClass ('dp' ) );
-require_once( $AppUI->getLibraryClass( 'PEAR/Date' ) );
+require_once( 'Date.php' );
 
 /**
  * The Project Class
@@ -68,4 +68,4 @@
 		}
 	}
 }
-?>
\ No newline at end of file
+?>
